synfig-studio/src/synfigapp/actions/valuedescmakeparenttoactive.cpp

```diff
@@ -0,0 +1,160 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file ValueDescMakeParentToActive.cpp
```

Comment:
I don't quite like this name:
1. The description makes clear it is related to Bone, mas the name doesn't
2. By the vocab/parameter list, it doesn't set the active bone as parent, but any bone you pass

So maybe it should be named "ValueDescSetParentBone" ?
I'm unsure about "ValueDesc" slice, but I don't recall how actions are named by now.

Reply:
This adds any bone we pass as the parent to active bone👀 so I guess the description is fine?

Reply:
It's not :) You have to pass which is the "active one". And it should be that way, IMO.

Another issue is the missing word "Bone" in class and action name.

Reply:
I agree with @rodolforg - it is better to rename this action to "ValueDescSetParentBone".
It would be even better to name it "ValueDescBoneSetParent", as we already have "ValueDescBoneLink" (that way the `valuedescbonesetparent.cpp` will be located together with `valuedescbonelink.cpp` in file list.

Following the same logic, it would be nice to rename "valuedesccreatechildbone.cpp/.h"  to "valuedescbonecreatechild.cpp/.h". But this should be done in separate PR. It would be nice to open a separate issue about that. @Aa20475 can you do that? ^__^

And of course, it's better to rename "active_bone" in param_vocab to "parent". ^__^

Reply:
Yes will do that! ^___^

Reply:
>And of course, it's better to rename "active_bone" in param_vocab to "parent". ^__^

Btw, isn't active bone the child here? :eyes:

Reply:
Please verify the changes.

Reply:
> Btw, isn't active bone the child here? :eyes:

You right, my mistake. ^__^

Reply:
It looks good now.